test/set-location.spec.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Behaviour = require('../apps/fbis-ccf/behaviours/set-location');
+
+describe('Set location behaviour', () => {
+
+  let req;
+  let res;
+  let SetLocation;
+  let testInstance;
+
+  beforeEach(() => {
+    req = {
+      sessionModel: {
+        set: sinon.stub().returns(undefined),
+        get: sinon.stub().returns(undefined)
+      },
+      query: {
+        'in-UK': undefined
+      }
+    };
+
+    res = {};
+  });
+
+  describe('getValues', () => {
+
+    class Base {
+      getValues() {}
+    }
+
+    beforeEach(() => {
+      SetLocation = Behaviour(Base);
+      testInstance = new SetLocation();
+    });
+
+    it('should set `in-UK` flag to false if `in-UK` query is false', () => {
+      req.query['in-UK'] = 'false';
```

Comment:
I'm confused, I thought req.query will return a JS object rather than a boolean value? when you pass in req.query for `fbis.gov.uk/?outside-UK` what will you get?

Reply:
I think this is where a lot of the confusion is coming from. I was assuming the URL would have a format something like ```fbis.gov.uk?in-UK=true```
or
```fbis.gov.uk?in-UK=false```
or just
```fbis.govuk```

I can refactor so that the outside UK URL is like you're suggesting here, will probably make the code clearer.